src/openforms/forms/tests/e2e_tests/test_form_designer.py

```diff
@@ -51,6 +53,15 @@ async def open_component_options_modal(page: Page, label: str):
 
 
 class FormDesignerComponentTranslationTests(E2ETestCase):
+    translation_map = {}
+    translations_data_path = "data[openForms.translations.{locale}]"
+    translations_literal_suffix = ""
+    translations_translation_suffix = "[translation]"
+    is_translations_order_fixed = False
```

Comment:
These constructs will become obsolete once the react-based formio builder is no longer behind a feature flag

Reply:
added TODOs!